asv.conf.json

```
@@ -0,0 +1,164 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "asv_sunpy_project",
```

should this not just be:
```suggestion
    "project": "sunpy",
```

---

No. ASV clones sunpy core repo into the directory named as this value in config. When I tried "sunpy" there was an error cause there is a "sunpy" folder, already. So there can be any name you want, but not "sunpy" as long as there is "sunpy" folder in core repo.

---

Ok, I would suggest "sunpy-benchmarks" or something then.

---

Good